ONBOARDING.md

```diff
@@ -1,50 +1,69 @@
 # Node.js build WG onboarding
 
-This document outlines what things future members of the build WG should be
-acquainted with. It might be expanded into additional documentation down
-the road.
+This document is an outline of the things we tell new Build WG members at their
+onboarding session.
 
-### General information
+### Before the onboarding session
 
-  - Responsibilities
-  - Members/"who does what"/interest areas
-  - Technologies and software in use
+* Email the new member and obtain their public GPG key
+* Add them to the `test` group in `nodejs-private/secrets`
+* Add them to the `nodejs/build` and `nodejs-private/build` GitHub teams
+* Ask them to start looking over `services.md` and `jenkins-guide.md`,
+  and to try and have any questions ready
+* Ask the onboardee to install:
+  * Ruby and the `dotgpg` gem
+  * Python 2.7 and Ansible
 
+### General information
 
-### CI/Jenkins
+* Responsibilities: The Build WG manages everything in
+  [`doc/services.md`](doc/services.md)
+* Members/"who does what"/interest areas
+* Technologies and software in use
 
-  - `nodejs/build` ci access
-  - CI and CI-Release
-  - Setting up a node
-  - `iptables`/infra group
-  - Jobs
+### CI/Jenkins
 
+* Public CI (ci.nodejs.org)
+  * `nodejs/jenkins-admins` are administrators
+  * Tests Node.js itself, and several other Foundation projects
+* Release CI (ci-release.nodejs.org)
+  * `nodejs/jenkins-release-admins` are administrators
+  * What creates and publishes all releases of Node.js
+* General trend is try and use Jenkins pipelines as much as possible,
```

Comment:
I'd add "where it does not make checking the results harder for authors". While pipelines are great to help us maintaining the jobs, I'd avoid using them in anything below `node-test-pull-request` as they can be very confusing for infrequent collaborators or fist time contributors. https://github.com/nodejs/build/issues/838#issuecomment-330717696

Reply:
ditto on this, I think working like "General trend is toward using Jenkins pipelines where they decrease complexity but don't negatively impact on usability for Node.js collaborators."